packages/editor-ui/src/views/SignoutView.vue

```diff
@@ -22,7 +22,7 @@ export default defineComponent({
 		async logout() {
 			try {
 				await this.usersStore.logout();
-				void this.$router.replace({ name: VIEWS.SIGNIN });
+				window.location.href = this.$router.resolve({ name: VIEWS.SIGNIN }).href;
```

Comment:
what would be the difference between this and 
```
void this.$router.replace({ name: VIEWS.SIGNIN }).finally(() => {
	this.$router.go(0);
});
```

Reply:
You see the login page flashing because first the router navigates there and after that, there is the reload